tools/internal_ci/linux/pull_request/grpc_bazel_on_foundry_dbg.sh

```diff
@@ -15,4 +15,3 @@
 
 EXTRA_FLAGS="-c dbg --test_timeout=300,450,1200,3600"
```

Comment:
`export UPLOAD_TEST_RESULTS=true` missing here?

Reply:
I don't think we want to collect results from pull request runs since it won't help in determining how flaky a test target is.

Reply:
ah, yes, you're right I didn't see the `pull_request` part.